src/App.tsx

```diff
@@ -5,16 +5,18 @@ import VolumetricPlayer from "./VolumetricPlayer";
 
 function App() {
   const [playerVisible, setPlayerVisible] = useState(false)
-
+  const paths = [
+    "http://172.160.10.156:3000/assets/brennan.drcs",
```

Comment:
This is the temporary path for the testing.

Reply:
This doesn't seem like something that should be committed; is there another way for you to do this?

Reply:
Hi @speigg, Thanks for the review.
I tried to use wildCapture.io assets, but I got the CORS error. so I used local assets.
Maybe I need to upload it to the online hosting server that has no CORS issue to fetch the assets.